### DISCUSSION TITLE
How does one add a Quick Fix to this project, specifically?

### DISCUSSION THREAD

**Discussion by hirehamir** [rust-lang/rust-analyzer#15155] [Q&A]
I'm aware of [Quick Fixes in the VSCode API](https://code.visualstudio.com/api/references/vscode-api#CodeActionKind.QuickFix), but I think a guide with exact steps on how to add a Quick Fix for this project specifically could encourage contributors.

**Comment by bjorn3** [comment]
https://rust-analyzer.github.io/blog/2020/09/28/how-to-make-a-light-bulb.html may be useful I think.
